go.mod

```diff
@@ -24,7 +27,7 @@ require (
 	github.com/pmezard/go-difflib v1.0.0
 	github.com/serenize/snaker v0.0.0-20201027110005-a7ad2135616e
 	github.com/sirupsen/logrus v1.8.1
-	github.com/spf13/afero v1.1.2
+	github.com/spf13/afero v1.2.2
```

Comment:
Is this required?

Reply:
Seems like a strange `go mod tidy` behaviour ... did [fixed it](https://github.com/grafana/xk6-websockets/commit/4ae924199850907c936b3b12bf556a0bef698736) in `xk6-websockets`. @oleiade  can you please bump to v0.1.1 :pray: :bow:

Reply:
I updated the dependency on `xk6-websockets` to `v0.1.1`, but the afero `1.2.2` dependency is still there. Investigating 👀

Reply:
@oleiade maybe you need to change this down to 1.1.2 manually now that it has once been set :thinking:

Reply:
Culprit seems to be `xk6-timers`:

```
> go mod graph | grep afero
go.k6.io/k6 github.com/spf13/afero@v1.2.2
github.com/grafana/xk6-redis@v0.1.1 github.com/spf13/afero@v1.1.2
github.com/grafana/xk6-timers@v0.1.0 github.com/spf13/afero@v1.2.2
github.com/grafana/xk6-websockets@v0.1.1 github.com/spf13/afero@v1.1.2
github.com/mstoykov/k6-taskqueue-lib@v0.1.0 github.com/spf13/afero@v1.1.2
github.com/spf13/afero@v1.2.2 golang.org/x/text@v0.3.0
```

Addressing it now 👍🏻

Reply:
`xk6-timers` depends on `k6-taskqueue-lib` which introduces the indirect dependency:

```
> go mod graph | grep afero
github.com/grafana/xk6-timers github.com/spf13/afero@v1.2.2
github.com/mstoykov/k6-taskqueue-lib@v0.1.0 github.com/spf13/afero@v1.1.2
github.com/spf13/afero@v1.2.2 golang.org/x/text@v0.3.0
go.k6.io/k6@v0.37.1-0.20220426072701-d105f5474bc3 github.com/spf13/afero@v1.1.2
```

The investigation continues 👀

Reply:
But [there](https://github.com/mstoykov/k6-taskqueue-lib/blob/main/go.mod#L24) the version is 1.1.2

Reply:
☝🏻 indeed, I realized my mistake while investigating 👍🏻 

The issue should be solved now 🙇🏻